datakyt/db/dump_db.py

```diff
@@ -0,0 +1,27 @@
+import subprocess
+
+
+def backup_postgres_db(host, database_name, port, user, password, dest_file):
+    """
+  Backup postgres db to a file.
+  """
+    try:
+        process = subprocess.Popen(
+            ['pg_dump',
+             '--dbname=postgresql://{}:{}@{}:{}/{}'.format(user, password, host, port, database_name),
+             '-Fc',
+             '-f', dest_file,
+             '-v'],
+            stdout=subprocess.PIPE
+        )
+        output = process.communicate()[0]
+        if int(process.returncode) != 0:
+            print('Command failed. Return code : {}'.format(process.returncode))
+            exit(1)
```

Comment:
I'm not sure we need this if-section in the `try` section. Once the process returned an error code, the `except` section will catch it and log the message. What do you think, @aartiukh ?

Reply:
It is a common approach to check whether an app return `SUCCESS = 0` or `FAILURE != 0`. I am okay with this check.